app/models/derivative.rb

```diff
@@ -53,6 +55,31 @@ def initialize(*args)
     self.managed = true
   end
 
+  alias_method :_width=, :width=
```

Comment:
Since these appear in master file as well, should they be pulled out into a concern?